src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -146,8 +146,41 @@ public override ExportResult Export(in Batch<Metric> batch)
                         }
                         else
                         {
-                            // TODO: Consider how/if to display buckets for exponential histograms.
-                            bucketsBuilder.AppendLine("Buckets are not displayed for exponential histograms.");
+                            var exponentialHistogramData = metricPoint.GetExponentialHistogramData();
+                            var scale = exponentialHistogramData.Scale;
+                            var factor = Math.Log(2) * Math.Pow(2, -scale);
+
+                            var offset = exponentialHistogramData.NegativeBuckets.Offset;
+                            List<string> negativeBucketCounts = null;
+                            foreach (var bucketCount in exponentialHistogramData.NegativeBuckets)
+                            {
+                                if (negativeBucketCounts == null)
+                                {
+                                    negativeBucketCounts = new List<string>();
+                                }
+
+                                var lowerBound = Math.Exp(offset * factor).ToString("F6", CultureInfo.InvariantCulture);
+                                var upperBound = Math.Exp(++offset * factor).ToString("F6", CultureInfo.InvariantCulture);
```

Comment:
Is this going to give the accurate IEEE 754 representation or there will be drifts due to precision losses or we don't know?

Reply:
I'm pretty confident this is absolutely imprecise 😆. I guess we need to consider the audience and use case we're trying to serve.

To me this verbose format is mainly for learning, debugging, non-production kind of use cases.

`F6` truncates things to 6 decimal places so that the width of the output is somewhat fixed and easier to read. I chose 6 simply because that's what the collector seems to display (might just be default formatting in Go).

Reply:
I essentially implemented the `LowerBoundary` reverse mapping method here 

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#all-scales-use-the-logarithm-function.

My understanding is that this is valid for both positive and negative scales. There is a note about the possibility that the maximum index is incorrectly computed as `+Inf`. I can verify.